apps/web-app/pages/[orgSlug]/settings/org/index.vue

```diff
@@ -84,7 +84,8 @@
     }
 
     uploadLoading.value = false;
-
+    refreshNuxtData('getUserSingleProfileNav');
```

Comment:
why is this here?
we only need to refresh the org profile since thats what we are changing